src/providers/postgres/qgspostgresconn.h

```diff
@@ -183,6 +183,12 @@ class QgsPostgresConn : public QObject
     Q_OBJECT
 
   public:
+    /* @param shared allow using a shared connection. Should never be
+     *        called from a thread other than the main one. An
+     *        assertion guards against that.
+     *
+     *  @deprecated Use QgsPostgresConnPool
```

Comment:
force-pushed new version of single-commit. Hadn't found a corresponding .sip (does it mean I don't have to bother about making bogus calls work rather than abort?)